app/docs/notes/Setu-Peppol.md

```diff
@@ -0,0 +1,29 @@
+
+# Sending SETU Messages via PEPPOL
```

Comment:
Don't use PEPPOL. with full caps. The official writing is `Peppol`. Find + replace in whole document